### PR TITLE
Update to GLib 2.68.0

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

File: gio/src/auto/dbus_proxy.rs [hunk 1]
```diff
@@ -251,6 +251,13 @@ impl DBusProxy {
     }
 }
 
+impl fmt::Display for DBusProxy {
+    #[inline]
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```
Comment (GuillaumeGomez) [inline gio/src/auto/dbus_proxy.rs:256]: Wow that's cool!

File: gio/src/auto/dbus_proxy.rs [hunk 2]
```diff
@@ -935,9 +942,3 @@ impl<O: IsA<DBusProxy>> DBusProxyExt for O {
         }
     }
 }
-
-impl fmt::Display for DBusProxy {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```
Comment (GuillaumeGomez) [inline gio/src/auto/dbus_proxy.rs:940]: Ah that was already here. :laughing:

Comment (sdroege) [reply gio/src/auto/dbus_proxy.rs:940]: Was just moved around for arbitrary reasons, yes

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
`Gio` seems to have issues.

**Comment by sdroege** [comment]
> `Gio` seems to have issues.

Already solved :)

**Review by bilelmoussaoui — APPROVED** [review]

**Pull request merged**